### PR TITLE
LIT 6576 6577

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Records for Dossier and Miracles of Mabaa Seyon. I constructed the miracles from a few sources, as only a part was edited by Budge. The list can be revised and corrected when more mss will be processed.

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]

File: new/LIT6576MiraclesMabaSeyon.xml [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+               <ref type="mss" corresp="EMML1779"/>, 
+               and <ref type="mss" corresp="ESmy024"/>.
+               The text follows the edition <bibl>
+                  <ptr target="bm:Budge1898MabaaSeyon"/>
+               </bibl> and Mss <ref type="mss" corresp="EMML3420"/> and <ref type="mss" corresp="ESmy024"/>.</note>
+
+               <div type="textpart" subtype="chapter" xml:id="Introduction">
+                <label>Introduction</label>
+                   <ab>
+                     በስመ፡ አብ፡<gap reason="omitted"/> ንጽሕፍ፡ እንከ፡ ድርሳነ፡ ተዝካረ፡ መድኃኒነ፡
```
Comment (thea-m) [inline new/LIT6576MiraclesMabaSeyon.xml:69]: I think what is meant here and below is gap reason="ellipsis"?

**Review by thea-m — CHANGES_REQUESTED** [review]

**Comment by eu-genia** [comment]
Denis, I am sorry, your last commit overwrites the corrections by me and Dorothea, because you did not refresh your branch with the commits we have pushed. Please take over the corrections we already did. 
In future, please always fetch and pull before committing any changes.
Thank you.

**Comment by eu-genia** [comment]
Take this file (the last pushed by Dorothea) https://raw.githubusercontent.com/BetaMasaheft/Works/9622d668f39d2f729d94acb48026849b9a7f046f/new/LIT6576MiraclesMabaSeyon.xml
and work with this one.

**Comment by DenisNosnitsin1970** [comment]
Which corrections? If you know that I am dealing with that and there is one open request, you do not need to intervene. The only correction I introduce was to substitute "omitted" for "ellipsis"

**Comment by thea-m** [comment]
This: https://github.com/BetaMasaheft/Works/pull/737/commits/9622d668f39d2f729d94acb48026849b9a7f046f
It is only the transcription of Taʾammərātihu, you can just repeat it in your version of the file.

**Comment by thea-m** [comment]
If you prefer, I can stop carrying out corrections of obvious typos in PRs. I just personally find it a waste of time for one person to state the presence of a type and then a second person to go and correct it, when the first person could do it in the same time as writing the comment.

**Comment by eu-genia** [comment]
Please check my commits before Dorothea's. I sent you a link to the latest version of the file which had my many corrections and hers.
(bm:bm-->bm:, emml->EMML, etc.)

**Review by eu-genia — APPROVED** [review]

**Comment by eu-genia** [comment]
I have now reintroduced the corrections

**Comment by DenisNosnitsin1970** [comment]
The corrections are in fact three or four, just recurrent. 
The record is not displayed in the BM.

**Comment by DenisNosnitsin1970** [comment]
It is displayed, but somehow wrong, without the content.

**Comment by eu-genia** [comment]
now is ok; i think the app was offline (i was editing the navigation) when you did the push so the webhook did not work

**Comment by DenisNosnitsin1970** [comment]
I still have a few corrections, I will creat a new branch, as usual.

**Pull request merged**